.github/workflows/test_and_deploy.yml

```
@@ -25,7 +25,7 @@ jobs:
     strategy:
       matrix:
         platform: [ubuntu-latest, windows-latest, macos-latest]
-        python-version: ["3.8", "3.9", "3.10", "3.11"]
+        python-version: ["3.8", "3.9", "3.10", "3.11", "3.12"]
```

```suggestion
        python-version: ["3.9", "3.10", "3.11", "3.12"]
```
napari no longer supports 3.8 (and soon 3.9), so I suggest dropping it here to keep the same number of CI runs